packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/ScmMethodProviderCollection.cs

```diff
@@ -40,8 +40,11 @@ public ScmMethodProviderCollection(InputOperation operation, TypeProvider enclos
             _cleanOperationName = operation.Name.ToCleanName();
             Client = enclosingType as ClientProvider ?? throw new InvalidOperationException("Scm methods can only be built for client types.");
             _createRequestMethod = Client.RestClient.GetCreateRequestMethod(Operation);
-            _isPaging = operation.Paging != null;
-            _paging = operation.Paging;
+            _isPageable = operation.Paging?.NextLink != null || operation.Paging?.ContinuationToken != null;
```

Comment:
is there really a case that we have the `Operation.Paging` is not null but nextlink and continuationtoken are both null? Or we are having this because some features we are not fully implemented as pageable APIs?

Reply:
Yes, this was a case in the Todo app.